src/intervals/functions.jl

```diff
@@ -4,61 +4,140 @@
 # Integer power:
 function ^{T<:Real}(a::Interval{T}, n::Integer)
```

Comment:
I thought the idea with powers was to just convert `Float64` directly to `BigFloat` intervals with 53 bits of precision, do the power there, and then round back.